src/Feed/NotifiedFeedType.php

```diff
@@ -54,13 +54,15 @@ public function getData(Feed $feed): array
 
             $result = [];
 
-            // Check that image is accessible, otherwise leave out the feed element.
+            // Check that image is available and accessible, otherwise leave out the feed element.
             foreach ($feedItems as $feedItem) {
-                $response = $this->client->request(Request::METHOD_HEAD, $feedItem['mediaUrl']);
-                $statusCode = $response->getStatusCode();
+                if (!empty($feedItem['mediaUrl'])) {
+                    $response = $this->client->request(Request::METHOD_HEAD, $feedItem['mediaUrl']);
```

Comment:
We should wrap http request in a try/catch, nor all failures will be covered by checking for a `200` status code. Not sure we need to do anything on failure execept exclude the `$feedItem` from `$result` as is done for non-200 status codes. (We could log it if we have a logger for the feed types, but I doubt it would have any practical use)